src/ensembl/src/shared/components/toolbox/toolbox-expandable-content/ToolboxExpandableContent.tsx

```diff
@@ -15,10 +15,9 @@
  */
 
 import React, { useState, useContext, ReactNode } from 'react';
-import classNames from 'classnames';
 import noop from 'lodash/noop';
 
-import CloseButton from 'src/shared/components/close-button/CloseButton';
+import Chevron from 'src/shared/components/chevron/Chevron';
```

Comment:
What if we used the `ShowHide` component for this?

Reply:
Like so:

```tsx
 
  return (
    <ShowHide label={props.label} onClick={handleClick} isExpanded={isExpanded} />
  );
```

Reply:
@azangru I wonder if I misunderstood this. Are you suggesting to use `ShowHide` in `ToolboxExpandableContent.tsx` or in `ZmenuAppLinks.tsx`?

Reply:
in `ToolboxExpandableContent.tsx`. Right where you are using the `Chevron`.

Reply:
@azangru Just pushed that change. FIgured out this morning that you might have said about the former. Yesterday, when I said I tried using `ShowHide`, I meant that for `ZmenuAppLinks.tsx`. Sorry for the confusion.